### DISCUSSION TITLE
how do i add new weeks?

### DISCUSSION THREAD

**Discussion by noskincz** [KadeArchive/Kade-Engine#855] [Q&A]
im trying to make a mod but idk how to make a new week

**Comment by Lucky-56** [comment]
here you go: https://kadedev.github.io/Kade-Engine/guides/weeks

**Comment by NotTheCoderGb** [comment]
https://kadedev.github.io/Kade-Engine/guides/weeks

or this 

https://github.com/KadeDev/Kade-Engine/blob/stable/docs/guides/weeks.md
